src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -176,32 +178,42 @@ export const Sidebar: React.FC<Props> = ({
     }
   }, [redeemCode, campaignData, refetchAll])
 
+  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
   const discountText = getDiscountText(textKeys)(
     redeemedCampaigns,
     offerData.cost.monthlyGross.currency,
   )
+  const clearDiscountCode = () => {
+    removeDiscountCode()
+      .then(() => {
+        const cookieStorage = new CookieStorage()
+        cookieStorage.setItem('_hvcode', '', {
+          path: '/',
+        })
+      })
+      .then(() => refetchAll())
+  }
 
-  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
+  const hasRedeemedCampaigns = redeemedCampaigns.length > 0
 
   return (
     <>
       <ReactVisibilitySensor partialVisibility onChange={setIsSidebarVisible}>
         {() => (
           <Wrapper>
             <Container>
-              <HeaderTop>
-                <HedvigSymbol />
+              {discountText && (
```

Comment:
Do we cover the cases correctly here in NO? Or do we require a `discountText` to always be present?

Reply:
Nice catch